layouts/_default/cve-feed.json

```diff
@@ -0,0 +1,24 @@
+{
+  "version": "https://jsonfeed.org/version/1.1",
+  "title": "Auto-refreshing Official CVE Feed",
+  "home_page_url": "https://kubernetes.io",
+  "feed_url": "https://kubernetes.io/docs/reference/issues-security/official-cve-feed/index.json",
+  "description": "Auto-refreshing official CVE feed for Kubernetes repository",
+  "authors": [
+    {
+      "name": "Kubernetes Community",
+      "url": "https://www.kubernetes.dev"
+    }
+  ],
+  "items": [
+  {{ range $i, $e := getJSON .Site.Params.cveFeedBucket }}
+    {{ if $i }}, {{ end }}
+      {
+        {{ T "cve_json_id" | jsonify }}: {{ .cve_id | jsonify }},
+        {{ T "cve_json_url" | jsonify }}: {{ .issue_url | jsonify }},
+        {{ T "cve_json_external_url" | jsonify }}: {{ .cve_url | jsonify}},
+        {{ T "cve_json_summary" | jsonify }}: {{ replace (.summary | jsonify ) "\\u003e" ">"  }}
```

Comment:
@sftim but adding it here did not fix things. Turns out it is a known issue: https://github.com/gohugoio/hugo/issues/7229 so I used the solution described in the issue

Reply:
I recommend adding a (Hugo) comment to explain why that replace is happening.

Reply:
Done. Hugo did not allow me to add comment in JSON file so added it in the corresponding markdown file